dev-tools/ls-apis/src/cargo.rs

```diff
@@ -0,0 +1,384 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Extract API metadata from Cargo metadata
+
+use crate::ClientPackageName;
+use anyhow::bail;
+use anyhow::{anyhow, ensure, Context, Result};
+use camino::Utf8Path;
+use camino::Utf8PathBuf;
+use cargo_metadata::Package;
+use cargo_metadata::{DependencyKind, PackageId};
+use std::collections::BTreeSet;
+use std::collections::{BTreeMap, VecDeque};
+
+/// Packages that may be flagged as clients because they directly depend on
+/// Progenitor, but which are not really clients.
+const IGNORED_NON_CLIENTS: &[&str] = &[
```

Comment:
Should this be part of `api-manifest.toml` as well?

Reply:
Done in 07dc35bdaa794b26457aa91ee874015ecfb3657c.